lambdas/data-migration1/tests/test-rules.js

```diff
@@ -0,0 +1,399 @@
+/* eslint-disable max-len */
```

Comment:
I'd rather not disable this rule and fix the cases where we have lines that are too long